src/datatrove/pipeline/extractors/goose3.py

```diff
@@ -0,0 +1,45 @@
+# Goose3 extractor for DataTrove - extract text with goose3
+# mrfakename (www.mrfake.name)
```

Comment:
usually as a principle HuggingFace's libraries do not include individual contributor's copyright notices
Your contribution would be indicated on a changelog (and obviously live forever on the commit history) but not so much on the header of files